src/utils/hooks/useLoadReview.tsx

```diff
@@ -1,34 +1,126 @@
 import { useEffect, useState } from 'react'
-import { useGetReviewQuery } from '../generated/graphql'
+import { ApolloError } from '@apollo/client'
+import {
+  ApplicationResponse,
+  ReviewResponse,
+  TemplateElement,
+  useGetReviewQuery,
+} from '../generated/graphql'
+import { ApplicationElementStates, SectionElementStates } from '../types'
+import useLoadApplication from './useLoadApplication'
+import useGetResponsesAndElementState from './useGetResponsesAndElementState'
+import getPageElements from '../helpers/getPageElements'
 
 interface UseLoadReviewProps {
   reviewId: number
+  serialNumber: string
 }
 
-const useLoadReview = ({ reviewId }: UseLoadReviewProps) => {
+const useLoadReview = ({ reviewId, serialNumber }: UseLoadReviewProps) => {
+  const [applicationName, setApplicatioName] = useState<string>('')
```

Comment:
```suggestion
  const [applicationName, setApplicationName] = useState<string>('')
```
Typo ;)

Reply:
Thanks! Reviews really help solving those before it's too late.